docs/getting-started/install_turtles_operator.md

```diff
@@ -4,4 +4,40 @@ sidebar_position: 4
 
 # Install Rancher Turtles Operator
 
-This section needs to be filled out.
\ No newline at end of file
+A `Rancher Turtles` requires a connection to the `Rancher Manager` cluster. This can be achieved by:
+
+1. Installing it in the same cluster as the `Rancher Manager`.
+
+*Note: In future we will support different deployment topologies*
+
+Recommended path of installation for the operator is by using `Helm`. To install it in the cluster, a repository should be added first:
+
+```bash
+helm repo add turtles https://rancher-sandbox.github.io/rancher-turtles/
+helm repo update
+```
+and then it can be installed into the `rancher-turtles-system` namespace with:
+
+```bash
+helm install rancher-turtles turtles/rancher-turtles
+    -n rancher-turtles-system
+    --create-namespace --wait
+```
+
+### Install `Cluster API Operator` as a Helm dependency
```

Comment:
This should be the default instructions instead of the previous helm command. wdyt?

Reply:
+1, manual installation of CAPI operator can be listed as an alternative